crates/sqlbuiltins/src/functions/table/object_store.rs

```diff
@@ -56,6 +58,13 @@ pub struct CsvOptionReader;
 impl OptionReader for CsvOptionReader {
     type Format = CsvFormat;
 
+    const OPTIONS: &'static [(&'static str, DataType)] = &[
+        // Specify delimiter between fields. Default: ','
+        ("delimiter", DataType::Utf8),
+        // Try to read a header. Default: true
+        ("header", DataType::Boolean),
```

Comment:
We should probably be consistent here. In excel, we've been calling this `has_header`. I think `has_header` is a bit more explicit. I don`t have strong feelings either way as long as they are consistent.

Reply:
Will update